transport/ice/IceSession.cpp

```diff
@@ -1166,7 +1166,7 @@ void IceSession::CandidatePair::send(const uint64_t now)
         state = InProgress;
     }
     _transactions.push_back(transaction);
-    if (_transactions.size() > 10)
+    if (_transactions.size() > 8)
```

Comment:
Reduce pressure on ice transaction map in udp endpoint. Risk would be that "rtt" is so long that we have already shifted out a request 8 items back and the response cannot be routed. But the interval is increased when there is no response so it should be enough time for a response to match up with an older pnding request.